### DISCUSSION TITLE
Multirepo in version 1.2. How?

### DISCUSSION THREAD

**Discussion by andlf** [werf/werf#4923] [General]
I have gitlab comunity setup, using werf v1.2.163+fix1, and i need to push many images to one project repository. I use key use-custom-tag "%image%:$CI_BUILD_REF_NAME" toset git tag to image tag. 
  WERF_IMAGES_REPO_MODE="multirepo" and WERF_REPO_CONTAINER_REGISTRY="gitlab" not working, images pushed at root image at gitlab registry. How can i push image to /group/project/imagename:tag from werf converge command?

**Comment by alexey-igrychev** [comment]
werf 1.2 does not support the multi-repo layout and stores all artifacts only in one container repository. werf takes full responsibility for delivery artifacts, and the user should not think about it.

But if you really need it, you can use the [werf export](https://werf.io/documentation/v1.2/reference/cli/werf_export.html) command and store them whenever you want. In that case, taking care of the artifacts becomes your responsibility.
